doc/zero-to-binderhub/setup-binderhub.rst

```diff
@@ -263,6 +263,8 @@ Second, you will need to instruct BinderHub about the token URL::
 If you setup your own local registry using
 `native basic HTTP authentication <https://docs.docker.com/registry/deploying/#native-basic-auth>`__
 (htpasswd), you can set ``token_url`` to ``None``.
+You will also need to configure the kubernetes nodes to authenticate to your private registry.
+Instructions can be followed on the `official kubernetes documentation <https://kubernetes.io/docs/concepts/containers/images/#configuring-nodes-to-authenticate-to-a-private-registry>`_.
```

Comment:
I think the statement is a bit too specific, because what's really needed is that all pods working with the images needs to be authenticated to pull them, and builder pods needs to be authenticated to push to them.

To configure the pods to have access, we can make use of `imagePullSecrets`. I don't remember how this helm chart does it any more at this point, but if this chart provides such configuration, that is also an option.

See https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod for details about this.

Reply:
I suggested this change because in my case it was by far the easiest solution (for an already spawned cluster). But if you think it is not necessary we can close this PR.